tests/cases/rest_decorator_test.py

```diff
@@ -19,16 +19,20 @@
 
 import json
 import os
+import requests
```

Comment:
Do we need to add requests to the requirements-dev.txt file (it is in the client/python/requirements.txt file)?